js/views/userPageVw.js

```diff
@@ -505,7 +504,7 @@ module.exports = Backbone.View.extend({
 
     $.ajax({
       type: "DELETE",
-      url: self.item.get('server') + "contracts/?id="+ self.item.get('id'),
+      url: self.item.get('server') + "contracts/?id="+ self.item.get('id'), //TODO: Change from GET to POST for safe methods, http://restcookbook.com/HTTP%20Methods/idempotency/
```

Comment:
Please read the provided link to http://restcookbook.com/HTTP%20Methods/idempotency/ - it explains why you should not send destructive methods by GET.